src/vm/jitinterface.cpp

```diff
@@ -1945,6 +1945,62 @@ CEEInfo::getClassSize(
     return result;
 }
 
+unsigned
```

Comment:
How about function header and interior comments here?

Reply:
Done.